src/main/java/uk/gov/hmcts/reform/cwrdapi/util/CaseWorkerConstants.java

```diff
@@ -90,7 +90,8 @@ private CaseWorkerConstants() {
     public static final String RECORDS_SUSPENDED = "%s record(s) suspended";
 
     public static final String DOMAIN_JUSTICE_GOV_UK = "justice.gov.uk";
-    public static final String USER_NAME_PATTERN = "^[\\w!#$%&’*+/=?`{|}~^-]+(?:\\.[\\w!#$%&’*+/=?`{|}~^-]+)*";
+    public static final String USER_NAME_PATTERN = "^[A-Za-z0-9]+[\\w!#$%&'’.*+/=?`{|}~^-]+"
+        + "(?:\\.[\\w!#$%&’*+/=?`{|}~^-]+)*";
```

Comment:
Do we really need to allow that many special characters, including '%'? It can possibly lead to SQL injection.
The team can provide the inputs as well.

Reply:
Hi Nitish, as part of this ticket it looks like we have decided to use the same regex expression for email as we currently have in PRD, so I have just copied that, but if this makes it vulnerable to SQL injections then we should definitely change it.

Reply:
Okay. Let's use the same regex as PRD for now, but we can request the BA to create a ticket for improving the regex.
This can be picked up in future sprints.